test/sensor/camera/test_camera.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import torch
+
+from kornia.geometry.vector import Vector3
+from kornia.image import ImageSize
+from kornia.sensor.camera import CameraModel, CameraModelType
+from kornia.testing import BaseTester
+
+
+class TestPinholeCamera(BaseTester):
```

Comment:
Can we have a compile/dynamo test here? Eg https://github.com/kornia/kornia/blob/master/test/filters/test_median.py#L101

Reply:
Does this only work for nn.Module ? Getting errors on this one.
```
test/sensor/camera/test_camera.py:54: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../.local/lib/python3.9/site-packages/torch/__init__.py:1441: in compile
    return torch._dynamo.optimize(backend=backend, nopython=fullgraph, dynamic=dynamic, disable=disable)(model)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <torch._dynamo.eval_frame.OptimizeContext object at 0x7f33a54af370>
fn = CameraModel(ImageSize(height=tensor([29., 24., 14., 93.,  9.], device='cuda:0', dtype=torch.float64), width=tensor([21...5890, 0.9594, 0.2872, 0.9332],
        [0.2920, 0.4996, 0.9555, 0.9751]], device='cuda:0',
       dtype=torch.float64))

    def __call__(self, fn):
        fn = innermost_fn(fn)
        # Optimize the forward method of torch.nn.Module object
        if isinstance(fn, torch.nn.Module):
            mod = fn
            new_mod = OptimizedModule(mod, self)
            # Save the function pointer to find the original callable while nesting
            # of decorators.
            new_mod._torchdynamo_orig_callable = mod.forward
            return new_mod
    
>       assert callable(fn)
E       AssertionError

../../.local/lib/python3.9/site-packages/torch/_dynamo/eval_frame.py:182: AssertionError
```

Reply:
oh, true, if I'm not wrong compile work for functions (callable) and for classes jut for Modules